src/chart/tree/TreeView.js

```diff
@@ -150,16 +152,15 @@ export default echarts.extendChartView({
 
     _updateViewCoordSys: function (seriesModel) {
         var data = seriesModel.getData();
-        var points = [];
-        data.each(function (idx) {
+        !this._points.length && data.each(idx => {
```

Comment:
We don't support arrow function in our build tool. Please store `var that = this;` outside the function instead of using arrow functions.
@100pah Please help check if storing `_points` to `this` is the correct way to do this.